src/replication.thrift

```diff
@@ -180,8 +180,9 @@ struct group_check_request
     // their WALs after this decree.
     5:optional i64          confirmed_decree;
 
-    // Used to deliver child gpid during partition split
-    6:optional dsn.gpid     child_gpid;
+    // Used to deliver child gpid and meta_split_status during partition split
+    6:optional dsn.gpid     child_gpid; 
```

Comment:
How primary knows its child_gpid? I think it certainly can calculates out on its own gpid.
Such as parent_gpid(1, 4) => child_gpid(1, 4+partition-count)
If primary can, the secondary can too.

Reply:
Primary calculate child_gpid in `trigger_primary_parent_split`, if it found the partition_count sync from meta server is double of its own value: 
https://github.com/XiaoMi/rdsn/blob/d850ee74917e1145fc6125bb174ccf28233dc3fe/src/replica/split/replica_split_manager.cpp#L1136-L1140
It is a safe solution to sync it to secondary, if any bug happened, such as duplicated split, secondary may get wrong child_gpid, leading dangerous result.